src/ocl/convolutionocl.cpp

```diff
@@ -266,18 +285,14 @@ static void EvaluateInvokers(Handle& handle,
             {
                 MIOPEN_LOG_I("Warning: skipping solver <" << sol.solver_id
                                                           << "> due to no workspace provided ("
-                                                          << sol.workspce_sz
-                                                          << " required)");
+                                                          << sol.workspce_sz << " required)");
```

Comment:
These formatting changes are not necessary. Our CI testing fails in such cases. Have you used `.clang-format` which resides in our repository?

Reply:
I'm using `.clang-format` in vscode now, but the format of code is always changed as above.

Reply:
What I am doing is running
```bash
find . -iname '*.h' \
    -o -iname '*.hpp' \
    -o -iname '*.cpp' \
    -o -iname '*.h.in' \
    -o -iname '*.hpp.in' \
    -o -iname '*.cpp.in' \
    -o -iname '*.cl' \
| grep -v -E '(build/)|(install/)' \
| xargs -n 1 -P $(nproc) -t clang-format-3.8 -style=file -i 2>/dev/null
```
from the root of the source tree.

Perhaps the clang-format you're using is not 3.8.